docs-md/index.md

```diff
@@ -1,21 +1,97 @@
 
-Blah.
+ksqlDB is an event streaming database purpose-built to help developers create
+stream processing applications on top of {{ site.aktm }}.
 
-What can I do with ksqlDB?
---------------------------
 
-Streaming ETL
+Why use ksqlDB to build stream processing applications?
+-------------------------------------------------------
+
+In today’s world, applications that leverage stream processing need a
+heavyweight architecture that requires integrating several distributed
+systems together. These architectures often include a fleet of connectors
+or agents for acquiring events from the outside world, durable storage to house
+these events, a map/reduce-like framework for processing the events, and a
+database to serve aggregations of events to applications. 
+
+Unfortunately, the parts don't fit together as well as you'd hope; all of these
+systems are complex, and each integration is a small project to figure out.
+It's like trying to build a car out of parts, but the parts come from different
+manufacturers who don't talk to each other.
+
+It shouldn't be this difficult to build stream processing applications.
+
+ksqlDB greatly reduces the operational complexity required to build stream
+processing applications, which enables you to build real-time systems without
+requiring significant time and overhead. It combines the power of real-time
+stream processing with the approachable feel of a relational database, through
+a familiar, lightweight SQL syntax. And because ksqlDB is natively powered by
+{{ site.aktm }}, it seamlessly leverages the underlying, battle-tested event
+streaming platform.
+
+What are the components of ksqlDB?
+----------------------------------
+
+These are the core categories that are foundational to building an app:
+collections, materialized views, and queries.
+
+### Collections
+
+Collections provide durable storage for sequences of events. ksqlDB offers
+two kinds of collections: streams and tables. Both operate under a simple
+key/value model. 
+
+- **Streams** are immutable, append-only collections. They're useful for
+  representing a series of historical facts. Adding multiple events with the
+  same key means that they are simply appended to the end of the stream.
+- **Tables** are mutable collections. They let you represent the latest version
+  of each value per key. They're helpful for modeling change over time, and
+  they're often used to represent aggregations.
 
-:   Apache Kafka is a popular choice for powering data pipelines. KSQL
+Because ksqlDB leverages {{ site.ak }} for its storage layer, creating a new
+collection equates to defining a stream or a table over a {{ site.aktm }}
+topic. You can declare a collection over an existing topic, or ksqlDB can
+create a new topic for the collection at declaration time. For more information,
+see [Collections](concepts/collections/overview.md).
+
+### Materialized Views
+
+Materialized views are derived representations of streams or tables. They
```

Comment:
All  tables are not materialized. but all materialized views are tables right? (atleast how we refer to it now. I understand they are purely derived and are very analogous to materialized views in databases). 

Do we want to say "Materialized Tables" instead? and clarify we are actually talking about tables here..

Reply:
Sounds reasonable. @MichaelDrogalis what do you think?

Reply:
Would prefer to keep the language since we're trying to be consistent in other places where we use the same. We can talk to Nick about making it a global change after the launch if we think that works better.